src/topology/sheaves/sheaf_condition/opens_le_cover.lean

```diff
@@ -315,6 +315,11 @@ begin
     rw ← (is_limit_opens_le_equiv_generate₂ F S hS).nonempty_congr, apply h },
 end
 
+lemma is_sheaf_iff_is_sheaf_pairwise_intersections :
```

Comment:
Can you please copy the docstring of `is_sheaf_iff_is_sheaf_pairwise_intersections'` here, and add a sentence there that points here?

Reply:
Done! Note that currently *opens_le_cover* imports *pairwise_intersections*, but the planned refactor will reverse the import and put this lemma back to *pairwise_intersections* though.